File: WS_EUREKABANK_SOAPJAVA_GR04/01.SERVIDOR/WS_EurekaBank_Server/src/main/java/ec/edu/monster/modelo/OperacionCuentaResponse.java
package ec.edu.monster.modelo;

import jakarta.xml.bind.annotation.XmlType;

@XmlType(name = "OperacionCuentaResponse")
public class OperacionCuentaResponse {

    private int estado;
    private double saldo;

    public OperacionCuentaResponse() {
    }

    public OperacionCuentaResponse(int estado, double saldo) {
        this.estado = estado;
        this.saldo = saldo;
    }

    public int getEstado() {
        return estado;
    }

    public void setEstado(int estado) {
        this.estado = estado;
    }

    public double getSaldo() {
        return saldo;
    }

    public void setSaldo(double saldo) {
        this.saldo = saldo;
    }
}

File: WS_EUREKABANK_SOAPDOTNET_GR04/01.SERVIDOR/WS_EB_DOTNET_SOAP_Servidor/ec.edu.monster.modelo/OperacionCuentaResponse.cs
using System;
using System.Xml.Serialization;

namespace WS_EB_DOTNET_SOAP_Servidor.ec.edu.monster.modelo
{
    [Serializable]
    [XmlRoot("OperacionCuentaResponse")]
    public class OperacionCuentaResponse
    {
        [XmlElement("estado")]
        public int Estado { get; set; }

        [XmlElement("saldo")]
        public decimal Saldo { get; set; } // dinero -> decimal
    }
}
